docs-website/plugins/custom-loaders/index.js

```diff
@@ -18,6 +18,33 @@ module.exports = function (context, options) {
                                 filename: 'build/[file].[hash]'
                             }
                         },
+                        {
+                            // Look for all require("*.toml") files
+                            test: /\.toml/,
```

Comment:
This is regex so could be written more concisely:

```suggestion
                            test: /\.(yaml|toml|ini)/,
```

Reply:
good eye! Thanks.  I was gonna change this myself and I had forgotten bc I had json in there and it didn't like it and I thought it didn't like the regex, but then i realized it was just the json it didn't like and forgot to change it back.